Places/Models/Itinerary.h

```diff
@@ -20,6 +20,9 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong) NSArray *placesToGo;
 @property (nonatomic, strong) NSNumber *estimatedCost;
 
+// array of String, each elem representing an activity, e.g. ["Created by Senem on July 21st, 2022", "Viewed by Iris on July 21st, 2022"]
+@property (nonatomic, strong) NSArray *activityHistory;
```

Comment:
Since the type is known, it might be a good idea to specify it so the compiler can warn about any type violations:

```
@property (nonatomic, strong) NSArray<NSString *> *activityHistory;
```

Also, as a potential improvement, maybe activity should be modeled as its own class containing the activity type, the user and the date? This would allow more flexibility for doing more than just displaying it as a string (stuff like sorting, filtering by user, adding the ability to tap on the user name to view the user profile, etc.)